openshift/templates/pvc-migrator-build.param

```diff
@@ -0,0 +1,9 @@
+#=========================================================
+# OpenShift template parameters for:
+# Component: .
+# Template File: ../openshift/templates/pvc-migrator-build.yaml
+#=========================================================
+NAME=pvc-migrator
+GIT_URI=https://github.com/BCDevOps/StorageMigration.git
+GIT_REF=master
+SOURCE_CONTEXT_DIR=docker
```

Comment:
The param file should go. The `init` function will create a `local` version to be used. Applies to all other param files as well.

Reply:
Agreed, `.param` files should not be checked in.